docs/core/extensions/dependency-injection-guidelines.md

```diff
@@ -3,7 +3,7 @@ title: Dependency injection guidelines
 description: Learn various dependency injection guidelines and best practices for .NET application development.
 author: IEvangelist
 ms.author: dapine
-ms.date: 09/23/2020
+ms.date: 10/29/2020
 ms.topic: guide
```

Comment:
nit: One thing I noticed is that down on line 30 there are four consecutive code blocks. This makes it hard to scan and read through. Looking at the first three, they are so similar that the differences between them aren't easily distinguishable. I would add a single sentence above each code block that informs the reader what they are looking at.